src/app/qrCodeScanner/page.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import React from 'react';
+import NavBar from '../../components/userComponents/navBar/navBar';
+import QrScanner from '../../components/userComponents/QrScanner/QrScanner';
+/**
+ * @returns qr code scanner page
+ */
+function App() {
+  return (
+    <div style={{ backgroundColor: '#FFFDF7', height: '100vh' }}>
+      <NavBar />
+      <div style={{ padding: '19px' }}>
```

Comment:
might want to change this number to something more regular (something that applies to most other styles, but idk what the team likes to do. also might want to refactor into tailwind style css